recipes/khronos-opencl-icd-loader/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "cb4acb99df3441e147bd859c20d9c3533ec95ce8" %}
+{% set sha256 = "e1cc1697d33d8662a0f082603ac8192b5feaa3f97afd3b4ff929ed41fd2fea13" %}
+
+package:
+  name: khronos-opencl-icd-loader
+  version: 2016.10.12
+
+source:
+  fn: {{ version }}.zip
+  url: https://github.com/KhronosGroup/OpenCL-ICD-Loader/archive/{{ version }}.zip
+  sha256: {{ sha256 }}
+
+  patches:
+  # Khronos don't ship those headers, but they require them to build.
+  # See https://github.com/KhronosGroup/OpenCL-ICD-Loader/blob/master/inc/README.txt
+  - add-khronos-headers.patch
+  # Khronos doesn't have an install target in their CMake. Add one.
+  - add-cmake-install.patch
+
+build:
+  number: 0
+
```

Comment:
The comment below and the extra space seems unnecessary, but feel free to keep it if you disagree as that does not prevent us from merging this.